clients/python-wrapper/tests/utests/test_branch.py

```diff
@@ -121,8 +121,21 @@ def monkey_revert_branch(repo_name, branch_name, revert_branch_creation, *_):
             assert revert_branch_creation.ref == ref_id
             assert revert_branch_creation.parent_number == expected_parent  # default value
 
+        def monkey_get_commit(repo_name, ref_name, **_):
```

Comment:
We need the following additional tests for revert_branch:
1. Pass reference_id and not reference
2. Pass reference and not reference_id
3. Don't pass any
4. Pass both